x-pack/filebeat/module/mssql/log/test/test.log

```diff
@@ -0,0 +1,21 @@
+2019-05-03 09:01:09.99 Server      Microsoft SQL Server 2017 (RTM-CU13) (KB4466404) - 14.0.3048.4 (X64)
```

Comment:
Timestamps here don't include timezones, we should probably add the `convert_timezone` option.

Reply:
Sounds good! Although I'm not sure about how it works I tried my best to add it and it should be included now.